build.sh

```diff
@@ -38,5 +38,5 @@ python_dist_type_id=166
 
 curl -X POST "$packagecloud/$endpoint" \
   -F "package[distro_version_id]=$python_dist_type_id" \
-  -F "package[package_file]=@dist/$package-$version-$format" \
+  -F "package[package_file]=@dist/$package_$version_$format" \
```

Comment:
err, i don't this is the typo? isn't the problem in the format var? https://github.com/Addepar/dataflake.fakeldap/blob/5e7d30c7640f5cd821b0909b45ab4d72ff001eae/build.sh#L28

Reply:
?